tests/.gitignore

```diff
@@ -34,3 +34,6 @@ logs
 x86
 bcl-test/templates
 *.7z
+BundledResources/Info.plist
+EmbeddedResources/Info.plist
+fsharplibrary/Info.plist
```

Comment:
Same

Reply:
This is due to vsmac making changes. I wpuld blame VSMac for modifying files when it should not.